src/main/java/com/nike/cerberus/command/CerberusCommand.java

```diff
@@ -136,7 +155,8 @@ public String getRegion() {
                         EnvironmentalVarRegion;
 
         if (StringUtils.isBlank(calculatedRegion)) {
-            throw new IllegalArgumentException("Failed to determine environment, checked 'CERBERUS_CLI_REGION' env var and -r, --region command options, options must go before the command");
+            log.error("Failed to determine environment, checked 'CERBERUS_CLI_REGION' env var and -r, --region command options, options must go before the command");
+            calculatedRegion = Regions.DEFAULT_REGION.toString();
```

Comment:
Any chance Amazon might change this region? Would it be better to specifically set the region we want?